### PR TITLE
Welcoming event

### PR DESCRIPTION
**Pull request by IohannesIohannium**
- Adds a welcoming event, through summoning of a character named Paradox Game Converters Group (English because that's our main working language, for Catholic I had no idea)
- Adds an (untested) effect to turn pregnant women at start up actually pregnant
- Removes game start flags for achievements and historical 9th century settings

SIDENOTE: Welcoming event does not appear in Observer mode

(Character's appearance is random)
image

### CODE REVIEW

File: ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+			trigger_event = {
+				id = welcome.1
+				days = 0
+			}
+		}
+		# Pregnancy time!
+		every_living_character = {
+			limit = { has_trait = pregnant }
+			make_pregnant = {
+				father = scope:thefather
```
Comment (IhateTrains) [inline ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt:276]: Where is the `scope:thefather` set?

File: ImperatorToCK3/Data_Files/blankMod/output/events/welcome.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	trigger = { }
+	
+	sender = character:converter_team
+	
+	immediate = {
+		character:converter_team = { death = natural }
+	}
+
+	option = { name = welcome_ok }
+}
```
Comment (IhateTrains) [inline ImperatorToCK3/Data_Files/blankMod/output/events/welcome.txt:17]: Can't we just use the `character_event` type? The converter_team character won't be needed.

**Review by IhateTrains — CHANGES_REQUESTED** [review]

File: ImperatorToCK3/Data_Files/blankMod/output/history/characters/hellothere.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+converter_team = {
```
[the remark this answers is not in the document]
Comment (IohannesIohannium) [reply ImperatorToCK3/Data_Files/blankMod/output/history/characters/hellothere.txt:1]: Solved

File: ImperatorToCK3/Data_Files/blankMod/output/events/welcome.txt (hunk 2 above)
Comment (IohannesIohannium) [reply ImperatorToCK3/Data_Files/blankMod/output/events/welcome.txt:17]: It's more of a "letter" to the player; if we should change it to a character event, I'll change it

File: ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt (hunk 1 above)
Comment (IohannesIohannium) [reply ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt:280]: Solved

File: ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+			game_rule_inversed_gender_equality_effect = yes #todo fix
+		}
+
+		### GAME RULE: SEXUALITY DISTRIBUTION ###
+		if = {
+			limit = { NOT = { has_game_rule = sexuality_distribution_default } }
+			game_rule_sexuality_distribution_effect = yes
+		}
+
+		# Parts about achievements removed
```
[the remark this answers is not in the document]
Comment (IohannesIohannium) [reply ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt:265]: It does not work like that for *on action* files

File: ImperatorToCK3/Data_Files/blankMod/output/events/welcome.txt (hunk 2 above)
Comment (IhateTrains) [reply ImperatorToCK3/Data_Files/blankMod/output/events/welcome.txt:17]: Alright.

File: ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt [hunk 5]
```diff
@@ -0,0 +1,290 @@
+# Called from code after history generation
```
Comment (IhateTrains) [inline ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt:1]: Please add the CK3 version note.

Comment (IohannesIohannium) [reply ImperatorToCK3/Data_Files/blankMod/output/common/on_action/game_start.txt:3]: Done

**Review by IhateTrains — APPROVED** [review]

**Pull request merged**